tests/components/zwave_js/test_sensor.py

```diff
@@ -1029,11 +1029,7 @@ async def test_last_seen_statistics_sensors(
     entity_id = f"{NODE_STATISTICS_ENTITY_PREFIX}last_seen"
     entry = entity_registry.async_get(entity_id)
     assert entry
-    assert not entry.disabled
-
-    state = hass.states.get(entity_id)
-    assert state
-    assert state.state == "2024-01-01T12:00:00+00:00"
+    assert entry.disabled
```

Comment:
We should enable it after this assertion and check that the state works and is a timestamp.

Reply:
Example:

https://github.com/home-assistant/core/blob/414057d455a48fcebbbeadce16a5ecc45bf82bb9/tests/components/zwave_js/test_discovery.py#L268-L275